### PR TITLE
Clarify error message when attempting to ref a property

### PR DESCRIPTION
**Pull request by ArcadeMode**
Proposes a fix for #60807

### CODE REVIEW

File: src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs [hunk 1]
```diff
@@ -8828,6 +8828,67 @@ static void Main()
...
+ref var l2 = ref r[0];//OK
+_ = M(ref c.N);       //CS0206
+_ = M(ref r.N);       //OK
+_ = M(ref c[0]);      //CS0206
+_ = M(ref r[0]);      //OK
+_ = M2(out c.N);      //CS0206
+_ = M2(out r.N);      //OK
+_ = M2(out c[0]);     //CS0206
+_ = M2(out r[0]);     //OK
+//defenitions
```
Comment (jjonescz) [inline src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs:8851]:
```suggestion
//definitions
```

**Review by jjonescz — CHANGES_REQUESTED** [review]

File: src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs (hunk 1 above)
Comment (ArcadeMode) [reply src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs:8851]: Fixed 😊

**Review by cston — APPROVED** [review]

**Review by jjonescz — APPROVED** [review]

**Comment by cston** [comment]
@ArcadeMode, thanks for the contribution.

**Comment by ArcadeMode** [comment]
It was a small one but hopefully more substantial ones to come 😁

**Pull request merged**